### DISCUSSION TITLE
Does it run on Grasshopper with Rhino 6?

### DISCUSSION THREAD

**Discussion by robin-gdwl** [visose/Robots#81] [Q&A] [answered]
I am trying to get it to run on rhino 6. But I cant open any of the example files, I always get an error that it couldnt read the file. When I place the "Load robot system" component it also gives an error: `1. Solution exception:'>' is an unexpected token. The expected token is '='. Line 44, position 92.`

Im guessing this refers to [line 44 of Util.cs](https://github.com/visose/Robots/blob/e2d7f0037ac721ae55090290186196eb84f0eb70/RobotsGH/Util.cs#L44l)? 
I followed all the other steps of the installation (installing .NET and loading the 3dm and .xml into Documents/Robots) 
I also cant open the Bartlett.3dm file in rhino is this normal? it says that it is not a .3dm file.
Anything else I could check ?
Thanks in advance.

**Comment by robin-gdwl** [comment, marked as answer]
Okay nevermind I just downloaded the files wrong as described and solved in Issue #4

**Comment by visose** [comment]
There's no better feeling that solving a problem by yourself 😋

**Comment by robin-gdwl** [comment]
Indeed! Very nice plugin by the way you should post it on Food4Rhino.com to make it easier to find. I have been trying all the free plugins there (machina, scorpion, Robots.io) for weeks without much success. putting my hopes into this one now :D
